### PR TITLE
Service/application fixes #2

### LINKED ISSUE #2: Our HTML should be semantic
In order to form a more perfect HTML doc and avoid crazy rendering issues we should link up with Travis CI and perform a lint of all the files.

I also suggest adding a Makefile which will properly lint so the user doesn't have to submit a pull request just to have the linting run.

### PR DESCRIPTION
**Pull request by evilnick**
charms.md and charms-bundles.md, including some minor tidy ups.

### CODE REVIEW

**Comment by evilnick** [comment]
@juju/docs some more service/application changes

File: src/en/charms.md [hunk 1]
```diff
@@ -12,18 +12,20 @@ TODO: Add images
...
-of experts. Charms make it easy to reliably and repeatedly deploy services, 
+cloud-based applications. The charms publicly available in the online
+[charm store](authors-charm-store.html) represent the distilled DevOps knowledge
+of experts. Charms make it easy to reliably and repeatedly deploy applications, 
 then scale up as required with minimal effort.
 
-There are a variety of topics related to charms to be found in the left navigation 
-pane under **Charms**. The three main topics you should get to know well are:
+There are a variety of topics related to charms to be found in the left 
+navigation pane under **Charms**. The three main topics you should get to knowwell are:
```
Comment (degville) [inline src/en/charms.md]: typo: knowwell / know well

File: src/en/charms.md [hunk 2]
```diff
@@ -142,18 +146,18 @@ mediawiki site is 'live'.
...
 ### Exposing the site
 
-By default, Juju is very secure. Juju itself, and the services it deploys can 
-see the other services in your cloud, but nothing and nobody else can. This
-isn't much use for a web service we want users to connect to, but Juju can
-easily make these services public. With it's understanding of the underlying 
+By default, Juju is very secure. Juju itself, and the applications it deploys 
+can see the other applications in your cloud, but nothing and nobody else can. 
+This isn't much use for a web service we want users to connect to, but Juju can
+easily make these applications public. With it's understanding of the underlying
```
Comment (degville) [inline src/en/charms.md]: typo: it's (not your fault)

**Comment by degville** [comment]
Two tiny typos (see comments) and it's good to go. :sunny: :sunglasses:

**Comment by evilnick** [comment]
Thanks!

**Pull request merged**